soh/src/overlays/actors/ovl_En_Ge1/z_en_ge1.c

```diff
@@ -247,6 +265,10 @@ void EnGe1_KickPlayer(EnGe1* this, GlobalContext* globalCtx) {
             globalCtx->nextEntranceIndex = 0x3B4;
         }
 
+        if (gSaveContext.n64ddFlag) {
```

Comment:
Similar to Bria's other comment, do we want these always in rando? Or just with entrance rando

Reply:
This one is special, it does an ER check inside for the ER logic, but also makes it so child link with hookshot does not get put into jail causing a soft lock (technically SoH supports child equip swap for hookshot), so I figured this was a worthy QoL for rando in general.